pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -358,3 +361,4 @@ func newOauthClient(tokenSource oauth2.TokenSource) (*http.Client, error) {
 
 	return oauth2.NewClient(oauth2.NoContext, tokenSource), nil
 }
+}
```

Comment:
No, we need separate alpha client. I guess - I'm not the one who wrote this code. @bowei